pkg/package-server/provider/inmem.go

```diff
@@ -76,7 +76,7 @@ func NewInMemoryProvider(informers []cache.SharedIndexInformer, queueOperator *q
 }
 
 // parsePackageManifestsFromConfigMap returns a list of PackageManifests from a given ConfigMap
-func parsePackageManifestsFromConfigMap(cm *corev1.ConfigMap, catalogSourceName, catalogSourceNamespace, catalogSourcePublisher, catalogSourceDisplayName string) ([]packagev1alpha1.PackageManifest, error) {
+func parsePackageManifestsFromConfigMap(cm *corev1.ConfigMap, catsrc *operatorsv1alpha1.CatalogSource) ([]packagev1alpha1.PackageManifest, error) {
```

Comment:
Change the function signature to just accept the whole `CatalogSource` rather than individual fields.